### ISSUE TITLE
Built-in Ad Block function in opera

### ISSUE THREAD

**Issue by cqx931** [dhowe/AdNauseam#583]
Opera is introducing this feature:["Block ads to make pages load faster"](http://help.opera.com/opera/Mac/2308/en/private.html#adBlocker).
So in opera, shall we include this preference setting as part of the ad blocker checking?

<img width="761" alt="screen shot 2016-11-06 at 4 58 02 pm" src="https://cloud.githubusercontent.com/assets/3056154/20036392/76062652-a442-11e6-8c1c-5b6492104e19.png">

**Comment by dhowe** [comment]
Yes, if this is enabled, then adn will not work. So, perhaps we check this instead?

**Comment by cqx931** [comment]
I looked at apis supported in opera, but didn't find the place to check this particular setting.
https://dev.opera.com/extensions/apis/

**Comment by dhowe** [comment]
I don't see it either. Perhaps you can post the question on StackOverflow?

**Comment by cqx931** [comment]
http://stackoverflow.com/questions/40457726/how-to-detect-whether-the-built-in-ad-blocker-is-enabled-in-opera
